Kfilter/src/main/java/com/isaacudy/kfilter/KfilterView.kt

```diff
@@ -458,11 +489,22 @@ class KfilterView @JvmOverloads constructor(context: Context,
         }
 
         override fun onFling(e1: MotionEvent, e2: MotionEvent, velocityX: Float, velocityY: Float): Boolean {
+            val yDistance: Float = Math.abs(e1.getY() - e2.getY())
+            val velocityY1 = Math.abs(velocityY)
+            if (velocityY1 > 400 && yDistance > 400) {
+                return if (e1.getY() > e2.getY()) // bottom to up
+                    true
+                else
+                    true
```

Comment:
This `if` statement is not needed - we return true in all cases, so it could be simplified to ` if (velocityY1 > 400 && yDistance > 400)  return true`

Reply:
Changed